ccmlib/cluster.py

```diff
@@ -148,9 +149,15 @@ def add(self, node, is_seed, data_center=None):
         node._save()
         return self
 
-    def populate(self, nodes, debug=False, tokens=None, use_vnodes=False, ipprefix='127.0.0.', ipformat=None):
-        if self.ipprefix:
-            ipprefix = self.ipprefix
+    def populate(self, nodes, debug=False, tokens=None, use_vnodes=False, ipprefix=None, ipformat=None):
```

Comment:
one of the test you've run, run cassandra ? to see we didn't broke that ?

Reply:
I'm not sure the BYO ran cassandra cluster but there is no change in observable behavior (at least there shouln't be)

Reply:
@fruch validated with `migration_test:TTLWithMigrate.big_table_with_ttls_test`